### PR TITLE
Add campus profile links to gql schema

### PR DESCRIPTION
**Pull request by vinnyjth**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by vinnyjth** [comment]
@conrad-vanl This is ready for you to look at. It throws an error when trying to open the `mailto` link in the iOS Simulator, but it works on Android. I believe it should not throw on a real iOS device.

File: packages/WillowCreekApp/src/tabs/connect/ActionTable/index.js [hunk 1]
```diff
@@ -13,66 +13,52 @@ import {
...
+    {({ data }) => (
+      <WebBrowserConsumer>
+        {(openUrl) => (
+          <View>
+            <PaddedView style={{ paddingBottom: 0 }}>
+              <H5 padded>Connect</H5>
+            </PaddedView>
+            <TableView>
+              {get(data, 'currentUser.profile.campus.resources', [])
+                .filter(({ icon }) => !icon) // Resources with an icon show up in the action bar.
```
Comment (conrad-vanl) [inline packages/WillowCreekApp/src/tabs/connect/ActionTable/index.js]: neat. Although... I almost wonder if these should be separate nodes? 

I guess this is fine for now. In the future we might have icons for the things in the list below.

Comment (vinnyjth) [reply packages/WillowCreekApp/src/tabs/connect/ActionTable/index.js]: Yah I also struggled with the right way to split these up. Probably should end up as separate nodes, but struggled with the right naming so used the icon for now.
